core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly

```diff
@@ -37,6 +37,7 @@ THE SOFTWARE.
       <l:task href="${rootURL}/configureClouds" icon="symbol-cloud" permission="${app.SYSTEM_READ}"
               title="${app.hasPermission(app.ADMINISTER) ? '%Configure Clouds' : '%View Clouds'}"/>
       <l:task href="configure" icon="symbol-settings" permissions="${app.MANAGE_AND_SYSTEM_READ}" title="${%Node Monitoring}"/>
+      <l:task href="labels" icon="symbol-tag" permissions="${app.MANAGE_AND_SYSTEM_READ}" title="${%Labels} (${it.getAllLabels().size()})"/>
```

Comment:
Why is this link limited to manage and system_read? Nodes and labels are not restricted so this doesn't contain any  information that should be hidden from normal users. Only the configure of the labels needs protection.

Reply:
I see. It was just copy paste from line above.